src/ui/Views/Courses/Index.cshtml

```diff
@@ -1,36 +1,69 @@
-﻿@model OrganisationCourses
+﻿@model CourseListViewModel
 
 @{
     Layout = "_Layout";
 }
 
-<h1 class="heading-xlarge">
-    <span class="heading-secondary">@Model.OrganisationName</span>
-    Courses by accrediting provider
-</h1>
-
 <div class="grid-row add-double-bottom-margin">
     <div class="column-two-thirds">
-        <p class="lede">Multiple UCAS courses for the same subject are combined into a single subject page. For example, courses with and without a salary.</p>
-    </div>
-</div>
+        <h1 class="heading-xlarge">
+            @Model.Courses.OrganisationName
+        </h1>
+
+        <h2 class="heading-large">
+            Request access for someone
+        </h2>
 
+        <p>
+            You can <a href="https://goo.gl/forms/knDj4JYC06iiZDn72">request a DfE Sign-in account</a> for others who manage your courses.
+        </p>
 
-    @foreach (var providerCourse in Model.ProviderCourses)
-    {
+        <h2 class="heading-large">
+            Check your courses
+        </h2>
 
-        <h3 class="heading-medium">
+        <p>
+            We found @Model.TotalCount courses belonging to @Model.Courses.OrganisationName.
```

Comment:
This will say "1 courses" for a few providers. We should fix in a separate PR when we've shipped.